packages/playground/wordpress/src/rewrite-rules.ts

```diff
@@ -0,0 +1,11 @@
+import type { RewriteRule } from '@php-wasm/universal';
+
+/**
+ * The default rewrite rules for WordPress.
+ */
+export const wordPressRewriteRules: RewriteRule[] = [
+	{
+		match: /^\/(.*?)(\/wp-(content|admin|includes).*)/g,
```

Comment:
Should this only match a single path segment? Or not? @bgrgicak 

```suggestion
match: /^\/([^/]+?)(\/wp-(content|admin|includes).*)/g,
```

Reply:
No, this should work with single and multiple paths. For example `/scope:0.1/subsite/` and `/subsite/`.

Reply:
👍 what about greedy vs non-greedy? E.g. how would this be handled `/scope:03938/subsite/wp-content/plugins/myplugin/wp-content/image.jpg`?

Reply:
>  E.g. how would this be handled /scope:03938/subsite/wp-content/plugins/myplugin/wp-content/image.jpg?

It would still work as expected and resolve as `/wp-content/plugins/myplugin/wp-content/image.jpg`.


![Screenshot from 2024-03-22 10-57-41](https://github.com/WordPress/wordpress-playground/assets/1199991/cc6fe912-da22-4100-9895-0b0edbfebbf3)

Reply:
Awesome, thank you for your patience! :-)

Reply:
This has been a great learning experience for me. I finally feel comfortable with Regex.